### PR TITLE
Add Bitcoin Cash to Any Signer

### PR DESCRIPTION
**Pull request by EnoRage**

### CODE REVIEW

File: src/Any/Signer.cpp [hunk 1]
```diff
@@ -201,6 +201,8 @@ TW::Any::Proto::SigningOutput TW::Any::Signer::sign() const noexcept {
         }
         break;
     }
+    case TWCoinTypeBitcoinCash:
+    case TWCoinTypeLitecoin:
```
Comment (catenocrypt) [inline src/Any/Signer.cpp]: Is Litecoin also included on purpose?

File: tests/Any/SignerTests.cpp [hunk 2]
```diff
@@ -216,7 +243,7 @@ TEST(Signer, NetworkNotSupported) {
     auto input = Proto::SigningInput();
     input.set_private_key("c9b0a273831931aa4a5f8d1a570d5021dda91d3319bd3819becdaabfb7b44e3b");
     input.set_transaction(transaction);
-    input.set_coin_type(TWCoinTypeBitcoinCash);
+    input.set_coin_type(TWCoinTypeZcash);
```
Comment (catenocrypt) [inline tests/Any/SignerTests.cpp]: Why is coinType changed here?

**Comment by catenocrypt** [comment]
Next time start from a new branch off recent master :)   (lots of unrelated commits are shown in the history of this PR)

**Comment by hewigovens** [comment]
Clean pr please

File: tests/Any/SignerTests.cpp (hunk 2 above)
Comment (EnoRage) [reply tests/Any/SignerTests.cpp]: Because it is supported right now, this test will fail otherwise

**Pull request closed without merging**